### ISSUE TITLE
can't install SDL

### ISSUE THREAD

**Issue by intelligencer** [Linuxbrew/linuxbrew#210]
```
dhc@dnb-smn-1:~$ brew install sdl
==> Downloading https://www.libsdl.org/release/SDL-1.2.15.tar.gz
Already downloaded: /home/dhc/.cache/Homebrew/sdl-1.2.15.tar.gz
==> Downloading http://bugzilla-attachments.libsdl.org/attachment.cgi?id=1320
Already downloaded: /home/dhc/.cache/Homebrew/sdl--patch-3137feb503a89a8d606405373905b92dcf7e293b.cgi
==> Downloading http://bugzilla-attachments.libsdl.org/attachment.cgi?id=1324
Already downloaded: /home/dhc/.cache/Homebrew/sdl--patch-08c19f077f56217fd300db390bca4c1a0bee0622.cgi
==> Patching
patching file src/video/quartz/SDL_QuartzVideo.h
patching file src/video/quartz/SDL_QuartzVideo.h
==> ./configure --prefix=/home/dhc/.linuxbrew/Cellar/sdl/1.2.15 --disable-nasm --without-x
==> make install
Error: inreplace failed
/home/dhc/.linuxbrew/Cellar/sdl/1.2.15/bin/sdl-config:
  expected replacement of "prefix=/usr/local" with "prefix=/home/dhc/.linuxbrew/Cellar/sdl/1.2.15"
dhc@dnb-smn-1:~$ 
```

**Comment by rwhogg** [comment]
@intelligencer were you able to successfully install sdl?

**Comment by sjackman** [comment]
Fixed by Homebrew/homebrew#34721
